projects/docs/src/app/docs-wrapper/docs-wrapper-routing.module.ts

```diff
@@ -0,0 +1,143 @@
+import { NgModule } from "@angular/core";
+import { Routes, RouterModule } from "@angular/router";
+import { DocsWrapperComponent } from "./docs-wrapper.component";
+import components from "../../assets/components-list.json";
+
+function getComponentPageRoutes(): Routes {
```

Comment:
So this dynamic import doesn't work anymore?

Reply:
removed